src/cli/commands/check.js

```diff
@@ -142,30 +144,27 @@ async function integrityHashCheck(
     reporter.error(reporter.lang(msg, ...vars));
     errCount++;
   }
+  const integrityChecker = new InstallationIntegrityChecker(config);
 
   const lockfile = await Lockfile.fromDirectory(config.cwd);
   const install = new Install(flags, config, reporter, lockfile);
 
   // get patterns that are installed when running `yarn install`
+  // TODO measure slowdown here
```

Comment:
This is a reminder for myself, I noticed check got slower in the last month.
Probably one of the following 2 commands to blame